python/lsst/meas/deblender/baselineUtils.cc

```diff
@@ -0,0 +1,123 @@
+template <typename ImagePixelT, typename MaskPixelT = lsst::afw::image::MaskPixel,
+          typename VariancePixelT = lsst::afw::image::VariancePixel>
+void declareBaselineUtils(py::module& mod, const std::string& suffix) {
+    using MaskedImageT = lsst::afw::image::MaskedImage<ImagePixelT, MaskPixelT, VariancePixelT>;
+    using ImagePtrT = std::shared_ptr<lsst::afw::image::Image<ImagePixelT>>;
+    using FootprintPtrT = std::shared_ptr<lsst::afw::detection::Footprint>;
+
+    py::class_<BaselineUtils<ImagePixelT, MaskPixelT, VariancePixelT>> cls(
+            mod, ("BaselineUtils" + suffix).c_str());
+    cls.def_static("symmetrizeFootprint",
+                   &BaselineUtils<ImagePixelT, MaskPixelT, VariancePixelT>::symmetrizeFootprint, "foot"_a,
+                   "cx"_a, "cy"_a);
+    // The C++ function returns a std::pair return value but also takes a referenced boolean
+    // (patchedEdges) that is modified by the function and used by the python API,
+    // so we wrap this in a lambda to combine the std::pair and patchedEdges in a tuple
+    // that is returned to python.
+    cls.def_static(
+            "buildSymmetricTemplate",
+            [](MaskedImageT const& img, lsst::afw::detection::Footprint const& foot,
+               lsst::afw::detection::PeakRecord const& pk, double sigma1, bool minZero, bool patchEdges) {
+                bool patchedEdges;
+                std::pair<ImagePtrT, FootprintPtrT> result;
+
+                result = BaselineUtils<ImagePixelT, MaskPixelT, VariancePixelT>::buildSymmetricTemplate(
+                        img, foot, pk, sigma1, minZero, patchEdges, &patchedEdges);
+                return py::make_tuple(result.first, result.second, patchedEdges);
+            });
+    cls.def_static("medianFilter", &BaselineUtils<ImagePixelT, MaskPixelT, VariancePixelT>::medianFilter,
+                   "img"_a, "outimg"_a, "halfsize"_a);
+    cls.def_static("makeMonotonic", &BaselineUtils<ImagePixelT, MaskPixelT, VariancePixelT>::makeMonotonic,
+                   "img"_a, "pk"_a);
+    // apportionFlux expects an empty vector containing HeavyFootprint pointers that is modified
+    // in the function. But when a list is passed to pybind11 in place of the vector,
+    // the changes are not passed back to python. So instead we create the vector in this lambda and
+    // include it in the return value.
+    cls.def_static("apportionFlux",
+                   [](MaskedImageT const& img, lsst::afw::detection::Footprint const& foot,
+                      std::vector<typename std::shared_ptr<lsst::afw::image::Image<ImagePixelT>>> templates,
+                      std::vector<std::shared_ptr<lsst::afw::detection::Footprint>> templ_footprints,
+                      ImagePtrT templ_sum, std::vector<bool> const& ispsf, std::vector<int> const& pkx,
+                      std::vector<int> const& pky, int strayFluxOptions, double clipStrayFluxFraction) {
+                       using HeavyFootprintPtrList =
```

Comment:
Weird indentation on this line.

Overall, I think this would be a lot more readable if you formatted as:
```
cls.def_static(
    "apportionFlux",
    [](
        MaskedImageT const& img,
        afw::detection::Footprint const& foot,
        ...
    ) {
        using HeavyFootprintList = ...
    }
);
```
That is, put each arg on a new line and put each brace- or open-paren on a new line.

You can of course use clang-format instead (which will probably not make it look as nice, but it should at least fix up the indentation).